doc/reference/native-tokens/02-getting-started.md

```diff
@@ -0,0 +1,565 @@
+
+```bash
+cardano-cli address key-gen \
+    --verification-key-file pay.vkey \
+    --signing-key-file pay.skey
+```
+
+The code should output something similar to this:
+
+```bash
+$ cat pay.skey 
+{
+    "type": "PaymentSigningKeyShelley_ed25519",
+    "description": "Payment Signing Key",
+    "cborHex": "5820aed07e0b1ddd946da278ffb1f671cc5b24c8453e6b47c24b0a6b15d818444fe8"
+}
+$ cat pay.vkey 
+{
+    "type": "PaymentVerificationKeyShelley_ed25519",
+    "description": "Payment Verification Key",
+    "cborHex": "582031752dd50ffe7ed90ba136ea775dacd5113ff67d13001a25aac953f719aa1f92"
+}
+```
+
+4. Generate the payment address:
+
+```bash
+./cardano-cli address build \
+--payment-verification-key-file pay.vkey \
+--out-file pay.addr \
+--testnet-magic 1
+```
+
+This code produces the following payment address:
+
+```bash
+$ cat pay.addr 
+addr_test1vqvlku0ytscqg32rpv660uu4sgxlje25s5xrpz7zjqsva3c8pfckz
+```
+
+5. Check the balance of the payment address:
+
+```bash
+./cardano-cli query utxo --address addr_test1vqvlku0ytscqg32rpv660uu4sgxlje25s5xrpz7zjqsva3c8pfckz --testnet-magic 1
+```
+
+The response should show no funds:
+
+```bash
+                           TxHash                                 TxIx        Amount
+--------------------------------------------------------------------------------------
+```
+
+6. Fund the address and check again:
+
+```bash
+./cardano-cli query utxo --address addr_test1vqvlku0ytscqg32rpv660uu4sgxlje25s5xrpz7zjqsva3c8pfckz --testnet-magic 1
+
+                           TxHash                                 TxIx        Amount
+--------------------------------------------------------------------------------------
+b1ddb0347fed2aecc7f00caabaaf2634f8e2d17541f6237bbed78e2092e1c414     0        1000000000 lovelace
+```
+
+7. Export the protocol parameters to a file for later use:
+
+```bash
+cardano-cli  query protocol-parameters \
+--testnet-magic 1 \
+--out-file protocol.json
+```
+
+#### Start the minting process
+
+1. Create a policy:
+
+```bash
+
+mkdir policy
+
+cardano-cli address key-gen \
+    --verification-key-file policy/policy.vkey \
+    --signing-key-file policy/policy.skey
+
+
+touch policy/policy.script && echo "" > policy/policy.script 
+
+
+echo "{" >> policy/policy.script 
+echo "  \"keyHash\": \"$(./cardano-cli address key-hash --payment-verification-key-file policy/policy.vkey)\"," >> policy/policy.script 
+echo "  \"type\": \"sig\"" >> policy/policy.script 
+echo "}" >> policy/policy.script 
+
+cat ./policy/policy.script 
+{
+  "keyHash": "5805823e303fb28231a736a3eb4420261bb42019dc3605dd83cccd04",
+  "type": "sig"
+}
+```
+
+2. Mint the new asset:
+
+```bash 
+
+$ ./cardano-cli transaction policyid --script-file ./policy/policy.script 
+328a60495759e0d8e244eca5b85b2467d142c8a755d6cd0592dff47b
+```
+
+#### Build the raw transaction
+
+1. Use this code to build the raw transaction:
+
+```
+$ echo -n "melcoin" | xxd -ps 
+6d656c636f696e
+
+./cardano-cli transaction build-raw \
```

Comment:
Is there a reason we are using `build-raw` and not `build` in the documentation?